reference/manifest/supporturl.md

```diff
@@ -1,27 +1,44 @@
 
 # SupportUrl element
-Specifies the URL of a page that provides support information for your add-in.
-
- **Add-in type:** Content, Task pane, Mail
 
+## Description
```

Comment:
Let's remove the Description heading so that the summary/description immediately follows the title. Then make the Example the first H2 in the topic.